### PR TITLE
Setup event listeners

### PR DESCRIPTION
**Pull request by Wes-Isaac**
# Math magicians: events

In this pull request I did the following:

- Create a Button component.
- Create a Display component.
- Use the logic component provided to do calculations.
- Add event listeners for the calculator.

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Component } from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (Emmyn5600) [inline src/components/Button.js]: - [ ] Let us try to fix the eslint errors without disabling them.

File: src/components/Display.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { Component } from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
+class Display extends Component {
```
Comment (Emmyn5600) [inline src/components/Display.js]: - [ ] Let us try to fix the eslint errors without disabling them.

File: src/components/Calculator.js [hunk 3]
```diff
@@ -1,31 +1,48 @@
 import React from 'react';
+import Button from './Button';
+import Display from './Display';
+import calculate from '../logic/calculate';
 import './calculator.css';
 
 // eslint-disable-next-line react/prefer-stateless-function
 class Calculator extends React.Component {
+  constructor() {
```
Comment (Emmyn5600) [inline src/components/Calculator.js]: - [ ] Let us try to fix the eslint errors without disabling them.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi, @Wes-Isaac   

Great work on making the project requirements:clap:
You've done well implementing the project requirements, but there are still some issues which aren't been addressed yet 
 
## STATUS: CHANGES REQUESTED :recycle:

### Changes requested.

- [ ] Check the comment under the review.

 #### Optional suggestions
 
 - I would recommend using hooks instead of a class-based component. refer to this [article](https://blog.bitsrc.io/6-reasons-to-use-react-hooks-instead-of-classes-7e3ee745fe04) for more details.
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello, @Wes-Isaac  great job working on the required changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**